dart/lib/src/transport/http_transport.dart

```diff
@@ -39,8 +44,19 @@ class HttpTransport implements Transport {
   }
 
   @override
-  Future<SentryId> send(SentryEvent event) async {
-    final data = event.toJson();
+  Future<SentryId?> sendSentryEvent(SentryEvent event) async {
+    final envelope = SentryEnvelope.fromEvent(event, _options.sdk);
+    return await sendSentryEnvelope(envelope);
+  }
+
+  @override
+  Future<SentryId?> sendSentryEnvelope(SentryEnvelope envelope) async {
```

Comment:
not a big fan of the names but we don't have overloads in Dart other than optional params :(

maybe @bruno-garcia has ideas

Reply:
Yeah, that was the only reason i introduced the method names. :/